ikarus/finiteelements/mechanics/eas/eas2d.hh

```diff
@@ -9,10 +9,28 @@
 
 #pragma once
 
+#include <ikarus/finiteelements/mechanics/easvariants.hh>
 #include <ikarus/utils/tensorutils.hh>
 
 namespace Ikarus::EAS {
 
+/**
+ * \brief Dummy struct for displacement-based EAS elements, i.e. 0 enhanced modes
+ */
+template <typename GEO>
+struct Q1E0
+{
+  static constexpr int strainSize         = 3;
+  static constexpr int enhancedStrainSize = 0;
+  using MType                             = Eigen::Matrix<double, strainSize, enhancedStrainSize>;
+  using DType                             = Eigen::Matrix<double, enhancedStrainSize, enhancedStrainSize>;
+
+  Q1E0() = default;
+  explicit Q1E0(const GEO& /*geometry*/) {}
+
+  static auto calcM(const Dune::FieldVector<double, 2>& /*quadPos*/) { return MType::Zero(); }
```

Comment:
Maybe write a comment why we use auto here and return the Zero expression. Same in H1E0

Reply:
Furthermore, this can be merged into one struct. the strainSize can be derived from the Geo dimensions.